fix(curriculum): use correct pronouns on workshop projects (#56769)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-node-and-express/587d7fb0367417b2b2512bef.md

# --description--

> You can respond to requests with a file using the `res.sendFile(path)` method. You can put it inside the `app.get('/', ...)` route handler. Behind the scenes, this method will set the appropriate headers to instruct your browser on how to handle the file you want to send, according to its type. Then it will read and send the file. This method needs an absolute file path. It has been recommended for you to use the Node global variable `__dirname` to calculate the path like this:

Reviewer: ```suggestion
You can respond to requests with a file using the `res.sendFile(path)` method. You can put it inside the `app.get('/', ...)` route handler. Behind the scenes, this method will set the appropriate headers to instruct your browser on how to handle the file you want to send, according to its type. Then it will read and send the file. This method needs an absolute file path. It's recommended that you use the Node global variable `__dirname` to calculate the path like this:
```
